src/utils/numbers.ts

```diff
@@ -0,0 +1,6 @@
+export const isAMultipleOfFive = (number: number): boolean => {
```

Comment:
`isMultipleOfFive` conveys the same information, in general, try to avoid the `a` conjuction on function names. (isOddNumber instead of isAnOddNumber, for instance).